javascript/javascript1/week3/smart-ease.js

```diff
@@ -0,0 +1,149 @@
+const notes = [];
+function saveNote(content, id) {
+  notes.push({ content: content, id: id });
```

Comment:
There's a nice little js feature that lets you do this
```suggestion
  notes.push({ content, id });
```

JavaScript will handle the assignment for you so it's equivalent to what you have
```
notes.push({ content: content, id: id });

```

You only do this when the property has the same name as the variable you're assigning it to.